docs/source/use-cases.rst

```diff
@@ -395,12 +395,22 @@ Streaming data engineering
 
 A data engineer responsible for watching a data feed needs to scale out a
 continuous process.  They `combine dask.distributed with normal Python Queues`_ to
-produce a rudimentary but effective stream processing system.
+produce a rudimentary but effective stream processing system. Alternatively, futures can be 
+streamed beween different clients using `channels`_ .
 
 Because dask.distributed is elastic, they can scale up or scale down their
 cluster resources in response to demand.
 
+End user application development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dask and Distributed offer a 100% Python alternative to message queue based tools such as `Celery`_. 
```

Comment:
100% is a strong statement and I'm not sure that Dask can live up to it.  See http://matthewrocklin.com/blog/work/2016/09/13/dask-and-celery